lib/web_console/templates/style.css.erb

```diff
@@ -1,9 +1,22 @@
-#console { position: fixed; left: 0; bottom: 0; width: 100%; height: 148px; border: 1px solid; padding: 0 0 4px 0; overflow: none; margin: 0; background: none repeat scroll 0% 0% #333; z-index: 9999; }
-#console div.console-inner { font-family: monospace; font-size: 11px; height: 100%; overflow: auto; background: #333; border: 0; padding: 0; margin: 0; }
-#console div.console-prompt-box { color: #FFF; }
-#console pre.console-message { color: #1AD027; margin: 0; border: 0; white-space: pre-wrap; background-color: #333; padding: 0; }
-#console div.console-focus span.console-cursor { background: #FEFEFE; color: #333; font-weight: bold; }
-#console div.resizer { width: 100%; height: 4px; cursor: ns-resize; }
-#console input.clipboard { height: 0px; padding: 0px; margin: 0px; width: 0px; margin-left: -1000px; }
-#console span.console-prompt-label { display: inline; color: #FFF; background: none repeat scroll 0% 0% #333; border: 0; padding: 0; }
-#console pre.console-prompt-display { display: inline; color: #FFF; background: none repeat scroll 0% 0% #333; border: 0; padding: 0; }
+.console .pos-absolute { position: absolute; }
+.console .pos-fixed { position: fixed; }
+.console .pos-right { right: 0; }
+.console .border-box { box-sizing: border-box; }
+.console .layer { width: 100%; height: 100%; }
+.console .layer.console-outer { z-index: 1; }
+.console .layer.resizer { z-index: 2; }
+.console { position: fixed; left: 0; bottom: 0; width: 100%; height: 148px; padding: 0; margin: 0; background: none repeat scroll 0% 0% #333; z-index: 9999; }
+.console .console-outer { overflow: auto; padding-top: 4px; }
+.console .console-inner { font-family: monospace; font-size: 11px; width: 100%; height: 100%; overflow: none; background: #333; }
+.console .console-prompt-box { color: #FFF; }
+.console .console-message { color: #1AD027; margin: 0; border: 0; white-space: pre-wrap; background-color: #333; padding: 0; }
+.console .console-focus .console-cursor { background: #FEFEFE; color: #333; font-weight: bold; }
+.console .resizer { background: #333; width: 100%; height: 4px; cursor: ns-resize; }
+.console .console-actions { padding-right: 3px; }
+.console .console-actions .button { float: left; }
+.console .button { cursor: pointer; border-radius: 1px; font-family: monospace; font-size: 9px; width: 10px; height: 10px; line-height: 10px; text-align: center; color: #ccc; }
```

Comment:
Can we make the close button a bit bigger? Say `font-size: 13px, width: 14px; height: 14px; line-height: 14px;`, I tested on my machine and it looks reasonable.